abdulfatai360/banka
# #166257705 Fix signup error for client

#### What does this PR do?
Fix the error that occur during client signup

#### Description of Task to be completed?
- Have client's signup error fixed

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/75/head:fx-client-signup-166257705`
- Now that you have a copy of the branch, switch to it using  `git checkout fx-client-signup-166257705`
- Your directory will now be an exact copy of the PR
- Run `npm install` to install all project's dependencies onto your local machine

#### What are the relevant pivotal tracker stories?
[#166257705](https://www.pivotaltracker.com/story/show/166257705)
